src/components/shared/Card/Card.js

```diff
@@ -72,15 +72,18 @@ const Card = ({
         <span className="Card__not-available-text">Not available</span>
       ) : (
         <>
-          <span className="Card__number">{formatNumber(number)}</span>
-          {percentChange === null ? (
-            <span className="Card__percent">(--%)</span>
-          ) : (
-            <span className="Card__percent">
-              ({Math.round(percentChange) > 0 && "+"}
-              {Math.round(percentChange)}%)
-            </span>
-          )}
+          <span className="Card__number">
+            {title === "Counties Reporting" ? `${formatNumber(number)}%` : formatNumber(number)}
```

Comment:
This is not the solution.
`Card` component is the UI component and should just render provided props.
In addition: it is a very unreliable check as the title could be changed later.

I think the better way is to move `formatNumber` logic to utils and render here just `{number}`.

Reply:
It is already in utils. I ended up with another solution using flag `isNumberPercent`.